content/en/reference_tables/_index.md

```diff
@@ -191,6 +191,12 @@ The selected CSV is upserted into the table, meaning that:
 
 Once the table is saved, the upserted rows are processed asynchronously and updated in the preview. It may take up to 10 minutes for the update to complete.
 
+## Export a Reference Table
+
+To export a Reference Table you can "Query in DDSQL Editor" and then leverage [DDSQL Editor][7] to export to CSV, Dashboard and more.
+<img width="878" height="265" alt="image" src="https://github.com/user-attachments/assets/8d8975af-9242-4e82-ba5c-1a627a95b349" />
```

Comment:
Can we replace this with the standard syntax for images in the docs repo? For example:

`{{< img src="path/to/your/image-name-here.png" alt="Your image description" style="width:100%;" >}}`